local/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Setup NextCloud
```

Comment:
It is good to move to the doc folder and in the main README of the project to have a summary with links to the md files in the doc folder

Reply:
i don't know about that, I mean, this README is exclusive for the setup on this folder, so i think it is more simple to have it here